patch.sh

```diff
@@ -282,7 +282,9 @@ declare -A patch_list=(
     ["575.64"]='s/\xe8\xb5\x2f\xfe\xff\x85\xc0\x41\x89\xc4/\xe8\xb5\x2f\xfe\xff\x29\xc0\x41\x89\xc4/g'
     ["575.64.03"]='s/\xe8\xb5\x2f\xfe\xff\x85\xc0\x41\x89\xc4/\xe8\xb5\x2f\xfe\xff\x29\xc0\x41\x89\xc4/g'
     ["575.64.05"]='s/\xe8\xb5\x2f\xfe\xff\x85\xc0\x41\x89\xc4/\xe8\xb5\x2f\xfe\xff\x29\xc0\x41\x89\xc4/g'
-    ["580.76.05"]='s/\xe8\x81\x2e\xfe\xff\x41\x89\xc6\x85\xc0/\xe8\x81\x2e\xfe\xff\x41\x89\xc6\x29\xc0/g'
+    ["580.65.06"]='s/\xe8\x81\x2e\xfe\xff\x41\x89\xc6\x85\xc0\x0f\x85\xa6\x00\x00\x00\x4c/\xe8\x81\x2e\xfe\xff\x29\xc0\x41\x89\xc6\x90\x90\x90\x90\x90\x90\x4c/g'
```

Comment:
ah i see, so i got the change right but there is more than one match for the bytecode string i had.

thanks for looking into this, will merge this!